ingestion/src/metadata/profiler/profiler/sampler.py

```diff
@@ -165,14 +168,52 @@ def _partitioned_table(self) -> Query:
             self.table.__table__.c,
         )
 
-        if not self._partition_details.partitionValues:
-            sample = (
+        if (
+            self._partition_details.partitionIntervalType
+            == PartitionIntervalType.COLUMN_VALUE
```

Comment:
sorry broad question - does this Sampler handle the pandas too?

Reply:
it does not. Right now pandas sampler will be in `datalake_sampler.py`.